pkgs/top-level/aliases.nix

```diff
@@ -762,6 +762,7 @@ mapAliases {
   libgme = game-music-emu; # Added 2022-07-20
   libgnome-keyring3 = libgnome-keyring; # Added 2024-06-22
   libgpgerror = throw "'libgpgerror' has been renamed to/replaced by 'libgpg-error'"; # Converted to throw 2024-10-17
+  libguestfs-appliance = throw "'libguestfs-appliance' has been renamed to 'libguestfs-appliance-fedora-bin'; also consider using 'libguestfs-appliance-debian' instead";
```

Comment:
Renames usually are not throws in the beginning to allow a smoother migration.

Reply:
Converted it to a warning, thanks!